docs/guides/q-ctrl-performance-management.ipynb

```diff
@@ -29,88 +29,27 @@
-    "- (Optional) A collection of parameter values to bind the circuit against.\n",
-    "- (Optional) A dictionary of run options, such as the shot count.\n",
-    "\n",
-    "**Accepted observables formats:**\n",
-    "- A Pauli string: `\"XY\"`\n",
-    "- A dictionary - Pauli strings with coefficients: `{\"XY\": 0.5, \"YZ\": 0.3}`\n",
-    "- A list of Pauli strings: `[\"XY\", \"YZ\", \"ZX\"]`\n",
-    "- A list of Pauli strings with coefficients: `[(\"XY\", 0.5), (\"YZ\", 0.3)]`\n",
-    "- A nested list of Pauli strings: `[[\"XY\", \"YZ\"], [\"ZX\", \"XX\"]]`\n",
-    "- A nested list of Pauli strings with coefficients: `[[(\"XY\", 0.1), (\"YZ\", 0.2)], [(\"ZX\", 0.3), (\"XX\", 0.4)]]`\n",
-    "\n",
-    "### Estimator outputs\n",
-    "| Name   | Type                            | Description                                                                                      | Example             |\n",
-    "|--------|---------------------------------|--------------------------------------------------------------------------------------------------|---------------------|\n",
-    "| values | `valueslist[ArrayLike[float]]`  | The resultant list of expectation values corresponding to the list of input PUBs.               | `[0.43046065915270004]` |\n",
+    "The following image depicts the error suppression methods automated by Fire Opal Performance Management.\n",
     "\n",
-    "### Parameter formats\n",
-    "The Sampler and Estimator both accept parameters, which follow the same [array broadcasting rules](./primitive-input-output#broadcasting-rules) as the `QiskitRuntime` primitives. The following parameter formats are accepted:\n",
+    "![Visualization of the error suppression pipeline](/images/guides/qctrl-performance-management/error_suppression.svg)\n",
     "\n",
-    "| Format                                      | Example                                                                                           |\n",
-    "|---------------------------------------------|---------------------------------------------------------------------------------------------------|\n",
-    "| 0-d array (single binding)                  | `{\"a\": 4, (\"b\", \"c\"): [5, 6]}`                                                                    |\n",
-    "| Single array (last index for parameters)    | `{\"a[0]\", \"a[1]\", \"a[2]\": [0.1, 0.2, 0.3]}`                                                     |\n",
-    "| Multiple arrays (last index for parameters, flexible dimensions) | `{\"c\": np.ones((10, 10, 2)).tolist(), \"b\": np.zeros((10, 10)).tolist()}`                       |"
+    "The function offers two primitives, Sampler and Estimator, and the inputs and outputs of both follow the implemented spec for [Qiskit Runtime V2 primitives](../guides/primitive-input-output). See the Inputs and Outputs sections for detailed interface information."
```

Comment:
```suggestion
    "The function offers two primitives, Sampler and Estimator, and the inputs and outputs of both follow the implemented spec for [Qiskit Runtime V2 primitives](../guides/primitive-input-output), which provides detailed interface information."
```
Rewording so it doesn't sound like the [Inputs/Outputs page](https://docs.quantum.ibm.com/guides/primitive-input-output) has sections labeled Inputs and Outputs.

Reply:
My mistake, I just now understood that this refers to the Inputs and Outputs section of this page! Let's instead hyperlink to the section itself if we can.

Reply:
I added some words to help with clarity.